src/jit/emitxarch.cpp

```diff
@@ -2242,9 +2242,11 @@ static bool isX87InsWithNoOperands(instruction ins)
 #if FEATURE_STACK_FP_X87
     return (ins == INS_f2xm1 || ins == INS_fchs || ins == INS_fld1 || ins == INS_fld1 || ins == INS_fldl2e ||
             ins == INS_fldz || ins == INS_fprem || ins == INS_frndint || ins == INS_fscale);
-#else  // !FEATURE_STACK_FP_X87
+#elif defined(_TARGET_X86_)
+    return ins == INS_frndint;
```

Comment:
Not being all that familiar with the x87 parts of the emitter, I found this utterly confusing, since these *do* have operands, and, in fact, the same operand profile as the other intrinsics, which aren't listed here. Then I realized that it means no *explicit* operands - and the only reason the others aren't included here, AFAICT, is that they are excluded from the code below that calls this. A comment would be great (though I realize that it's not your issue).